portfolio/src/main/webapp/script.js

```diff
@@ -59,4 +82,32 @@ async function loadProjects(){
   for(var i =0; i<projects.length; i++){
       projectList.innerHTML += "<li> <a target= \"_blank\" href=\""+projects[i].url+"\">"+projects[i].name;
   }
-}
\ No newline at end of file
+}
+
+function requestTranslation() {
+        const text = document.querySelectorAll('.txt');
+        const languageCode = document.getElementById('language').value;
+        console.log(text);
```

Comment:
You might consider removing this log statements, I'm assuming they were there for debugging? Also similar comment about newlines here.